svelte.config.js

```diff
@@ -23,7 +23,7 @@ const config = {
       entries: ["*"],
     },
     paths: {
-      base: dev ? "" : "/knowledge-hub",
+      base: "",
     },
```

Comment:
I think we can remove this whole block since its the default

https://svelte.dev/docs/kit/configuration#paths

Then we're good to go :)